### PR TITLE
Removes some oldcode

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
This pr removes:
- The old specops console (no longer used)
- The old syndicate specops console (no longer used)
- The old salvage console (no longer used)
- Honksquads as a secrets button (The code has been majorly improved and they are now spawned with Gimmick Team)
- Bananium computer frames
- Uncut ruby

## Why It's Good For The Game
- For the consoles, these things are carryovers from ancient code, and use awful UIs. They also dont exist anywhere.
- Bananium computer frame required highly snowflaked construction and material fuckery, and isnt used
- The uncut ruby also did not exist anywhere and its git blame predates paradise itself.
- Honksquad raw code was awful, so its been refactored

## Changelog
:cl:
del: Removed some unused code
/:cl:

## Note

If I see a single "improve dont remove" comment on this PR, you will be shamed and told to improve it yourself. These features havnt been touched in over 8 years. Arguing with buzzwords gets you nowhere.

### CODE REVIEW

**Comment by SabreML** [comment]
RIP Honksquad, you will be missed.
(Even though nobody ever actually saw you)

**Comment by Carthusia** [comment]
I shall miss the honksquad...hopefully it comes back in a more refined form.

Honksquad v BoxSquad (Mimes)

Uncut ruby...ive seen that once, but why do i get runescape vibes from that item name lol

**Comment by Sinestia** [comment]
I remember being depressingly disappointed in old paradise because the wiki implied honk squad computers could be made but alas NO

10/10 rip out my trauma

**Comment by Sirryan2002** [comment]
destroy the old, make way for the new 😤

**Review by dearmochi — APPROVED** [review]

File: code/game/machinery/computer/buildandrepair.dm [hunk 1]
```diff
@@ -552,116 +544,3 @@
 	if(I.use_tool(src, user, 50, volume = I.tool_volume) && !state)
 		to_chat(user, "<span class='notice'>You deconstruct [src].</span>")
 		deconstruct(TRUE)
-
-
-
-/obj/structure/computerframe/HONKputer
-	name = "Bananium Computer-frame"
-	icon = 'icons/obj/machines/HONKputer.dmi'
-	base_mineral = /obj/item/stack/sheet/mineral/bananium
```
Comment (SabreML) [inline code/game/machinery/computer/buildandrepair.dm:561]: The `base_mineral` variable can be removed entirely, since it's only actually used here.

**Comment by Varghh** [comment]
Remove don't improve

**Review by Kyep — APPROVED** [review]
I find it hard to believe that honksquad went that long without being used. I question whether those dates are accurate.

That said... sure, why not. Honksquad is pretty poor quality code anyway.

**Comment by AffectedArc07** [comment]
> I find it hard to believe that honksquad went that long without being used. I question whether those dates are accurate.
> 
> That said... sure, why not. Honksquad is pretty poor quality code anyway.

There were some more recent ones, I was merely using those 2 snapshots to quantify how infrequently they are used. The same can be done with random spawn ins, without a mess of snowflake code. 

I can still keep them in if desired.

**Comment by Kyep** [comment]
Maybe make honksquad an admin outfit? That way it can still be sent using the (frankly better) gimmick team verb, but doesn't have a whole ton of its own useless snowflake code.

I feel like if this PR had "removes honksquad" in the title it would get buried in 👎 emotes. Enough players would feel like that is a loss, even if its almost never used. As it is, this PR is kinda sneaking under the radar for a lot of people.

**Comment by AffectedArc07** [comment]
> Maybe make honksquad an admin outfit? That way it can still be sent using the (frankly better) gimmick team verb, but doesn't have a whole ton of its own useless snowflake code.
> 
> 
> 
> I feel like if this PR had "removes honksquad" in the title it would get buried in 👎 emotes. Enough players would feel like that is a loss, even if its almost never used. As it is, this PR is kinda sneaking under the radar for a lot of people.

Good idea. I'll do that.

**Comment by AffectedArc07** [comment]
Update:

Honksquads are no longer removed, but they are now an outfit to be used with the gimmick team. The spawning has also been migrated to an outfit because its currently awful code.

**Comment by Kyep** [comment]
Good job.

**Pull request merged**